docs/index.yaml

```diff
@@ -3,7 +3,39 @@ entries:
   keda:
   - apiVersion: v1
     appVersion: 1.4.0
-    created: "2020-04-17T10:33:25.9773445-07:00"
+    created: "2020-04-21T16:43:41.810696+02:00"
+    description: Event-based autoscaler for workloads on Kubernetes
+    digest: 7b2b4e29f867361af87a60411b0a196416b04b2e20f4091d166263ea5195c355
+    home: https://github.com/kedacore/keda
+    icon: https://raw.githubusercontent.com/kedacore/keda/master/images/keda-logo-500x500-white.png
+    maintainers:
+    - email: ahmels@microsoft.com
+      name: Ahmed ElSayed
+    name: keda
+    sources:
+    - https://github.com/kedacore/keda
+    urls:
+    - https://kedacore.github.io/charts/keda-1.4.0.tgz
+    version: 1.4.0
+  - apiVersion: v1
+    appVersion: 1.3.0
+    created: "2020-04-21T16:43:41.8076969+02:00"
+    description: Event-based autoscaler for workloads on Kubernetes
+    digest: 163da9f24dbc38d4ccdfa3a8c4b08fa9f355b6a7af5afdebf5242348e4af59fb
+    home: https://github.com/kedacore/keda
+    icon: https://raw.githubusercontent.com/kedacore/keda/master/images/keda-logo-500x500-white.png
+    maintainers:
+    - email: ahmels@microsoft.com
+      name: Ahmed ElSayed
+    name: keda
+    sources:
+    - https://github.com/kedacore/keda
+    urls:
+    - https://kedacore.github.io/charts/keda-1.3.3.tgz
+    version: 1.3.3
```

Comment:
Actually I'm not sure if we should even ship this since it contains the Kubernetes resources but just for 1.3.0.

Either we need to go back to 1.3.1 branch and create it from there or just bite the bullet and ignore the fact that 1.3.2 was a release by accident and only ship 1.4?

Reply:
Yeah, I'd stick to the easier solution.

Reply:
Updated.